### PR TITLE
userEnabled PR #3 - Refactor from /manage to /users/manage

### PR DESCRIPTION
**Pull request by esarnb**
Applied all comments from previous PR review, to this latest commit. 
Tested and works the same way as before.

### CODE REVIEW

File: controllers/users-controller.js [hunk 1]
```diff
@@ -59,8 +59,66 @@ const userGet = (req, res) => {
     });
 };
 
+const getManage = (req, res) => {
+  const {
+    session: { isLoggedIn },
+  } = req;
+
+  console.log(`\n\n\n\n\nGET MANAGE\n\n\n\n\n`);
```
Comment (esarnb) [inline controllers/users-controller.js]: Need to remove console.log

File: controllers/users-controller.js [hunk 2]
```diff
@@ -59,8 +59,66 @@ const userGet = (req, res) => {
...
+  User
+    .findAll({
+      order: [['name', 'ASC']],
+    })
+    .then((data) => {
+      console.log(data);
```
Comment (esarnb) [inline controllers/users-controller.js]: Remove log

File: controllers/users-controller.js [hunk 3]
```diff
@@ -59,8 +59,66 @@ const userGet = (req, res) => {
...
+      const users = data.map((user) => user.get({ plain: true }));
+      const enabledUsers = users.filter((user) => user.isEnabled);
+      const disabledUsers = users.filter((user) => !user.isEnabled);
+      console.log(enabledUsers);
+      console.log(disabledUsers);
```
Comment (esarnb) [inline controllers/users-controller.js]: remove log

File: utils/index.js [hunk 4]
```diff
@@ -1,4 +1,5 @@
 /* eslint-disable global-require */
 module.exports = {
   getRoutes: require('./get-routes'),
+  clearErrors: require('./clear-errors'),
```
Comment (esarnb) [inline utils/index.js]: Remove leftover clearErrors

File: utils/clear-errors.js [hunk 5]
```diff
@@ -0,0 +1,7 @@
+const clearErrors = (error) => Object.keys(error)
+  .forEach((key) => {
+    // eslint-disable-next-line no-param-reassign
+    error[key].textContent = '';
+  });
+
+module.exports = clearErrors;
```
Comment (esarnb) [inline utils/clear-errors.js]: remove unused clearErrors files, since theres a utils.js in public

**Review by esarnb — COMMENTED** [review]
I need to fix these in next feature branch

**Review by SamMaxwell — CHANGES_REQUESTED** [review]
Instead of adding review comments to yourself, just make the changes you want or add TODOs into the code.

**Review by SamMaxwell — APPROVED** [review]

**Pull request merged**